website/search/elastic_search.py

```diff
@@ -576,8 +576,8 @@ def update_file(file_, index=None, delete=False):
         provider=file_.provider,
         path=file_.path,
     )
-    if file_.node.is_quickfiles:
-        node_url = '/{user_id}/quickfiles/'.format(user_id=file_.node.creator._id)
+    if target.is_quickfiles:
```

Comment:
Might be a good idea to use `getattr`, in case `target` isn't node. Shouldn't affect anything now, though, so this is good to merge.

Reply:
@sloria, completely agree; didn't change in this PR because I noticed the same thing a few lines above:
```
target.is_public or delete or target.is_deleted or target.archiving
```
and wasn't sure if there was an `isinstance(target, Node)` hiding somewhere else. 

It looks like there are several instances of this that will need to be updated if files are attached to non-node objects (users).